README.md

```diff
@@ -3,19 +3,15 @@
 [![Build Status](https://github.com/laminas/laminas-db/workflows/Continuous%20Integration/badge.svg)](https://github.com/laminas/laminas-db/actions?query=workflow%3A"Continuous+Integration")
 
 > ## 🇷🇺 Русским гражданам
-> 
+
```

Comment:
This is not a correct. markdown lint complained about trailing whitespaces in `> `